### PR TITLE
Remove fallback for fast call errors

### PR DESCRIPTION
**Pull request by littledivy**

### CODE REVIEW

File: ops/op2/dispatch_fast.rs [hunk 1]
```diff
@@ -323,36 +323,30 @@ pub(crate) fn get_fast_signature(
...
-        // TODO(mmastrac): We should allow an #[op] flag to re-perform slow calls without the error path when
-        // the method is performance sensitive.
+        #maybe_scope
 
-        // SAFETY: We guarantee that OpCtx has no mutable references once ops are live and being called,
-        // allowing us to perform this one little bit of mutable magic.
-        unsafe { #opctx.unsafely_set_last_error_for_ops_only(err); }
-        #fast_api_callback_options.fallback = true;
+        let err = err.into();
+        let exception = deno_core::error::to_v8_error(
```
Comment (lucacasonato) [inline ops/op2/dispatch_fast.rs:344]: This works now because we can now alloc in fast calls?

Comment (littledivy) [reply ops/op2/dispatch_fast.rs:344]: yes

**Comment by bartlomieju** [comment]
@littledivy can this PR can now be updated or are we waiting for V8 12.8 which should come out on Monday, July 22nd?

**Comment by littledivy** [comment]
It's crashing so I think we'll have to wait for V8 12.8

```
#
# Fatal error in ../../../../v8/src/api/api-inl.h, line 173
# Debug check failed: v8::internal::ValueHelper::IsEmpty(that) || IsJSReceiver(v8::internal::Tagged<v8::internal::Object>( v8::internal::ValueHelper::ValueAsAddress(that))).
#
#
#
#FailureMessage Object: 0x7ff4317fb280
==== C stack trace ===============================

    /home/runner/work/deno_core/deno_core/target/release/deps/deno_core-d4d659f5b998ad36(+0x3a3983) [0x5574aad36983]
    /home/runner/work/deno_core/deno_core/target/release/deps/deno_core-d4d659f5b998ad36(+0x3a529d) [0x5574aad3829d]
    /home/runner/work/deno_core/deno_core/target/release/deps/deno_core-d4d659f5b998ad36(+0x39f51e) [0x5574aad3251e]
    /home/runner/work/deno_core/deno_core/target/release/deps/deno_core-d4d659f5b998ad36(+0x39ef55) [0x5574aad31f55]
    /home/runner/work/deno_core/deno_core/target/release/deps/deno_core-d4d659f5b998ad36(+0x4704c0) [0x5574aae034c0]
    /home/runner/work/deno_core/deno_core/target/release/deps/deno_core-d4d659f5b998ad36(+0x37e06a) [0x5574aad1106a]
    /home/runner/work/deno_core/deno_core/target/release/deps/deno_core-d4d659f5b998ad36(+0x17d657) [0x5574aab10657]
    /home/runner/work/deno_core/deno_core/target/release/deps/deno_core-d4d659f5b998ad36(+0x266739) [0x5574aabf9739]
```

**Comment by littledivy** [comment]
Superseded by https://github.com/denoland/deno_core/pull/800

**Pull request closed without merging**